src/de/schildbach/pte/VrsProvider.java

```diff
@@ -334,9 +321,13 @@ public LocationWithPosition(Location location, Position position) {
JwAwyEpYJ6ZC1aaFP5lciqpIgJkWzzxHYPC39qJapf7Yi92+a52Tb9lskO9apqDhCOkQ4zW/Ch3isE4aHWdosvkZSKGYFjyJ+juCm6M11CDhqaJxGPsX2wGKfbc71sfzV18n7DLIlcYNdzOIKacF82YTCEfF8DNTiszE1r8gHzf1S918XthZ4tVQDhsUWefHlTYAZKtGO8/iApqxsex7XQeAyoMsEggQAaGcU4t0gWp8h14J8Sr0kKuWMiJg+kL7NHLf/HrhbduqhJyjce+s96Bxf4SBjNYyNMiFTZdxJIyKJgpB4RfTPKrp0guMRiiNycdr5vEGPF1A8+6MhcQWYc2gT21HjTXxxYH6QTo9/N0PZEBjKg9MErl0q75c1naOd+FopufclvTZQ5WYGxdonYy9196DZi2s07ZmTSy2F+tC2yZ0IkJufadoQYYECw8JZ7mTvmE0UwAuTfcrp/ePcUs9sb8dA7VsnvidK5xFxJ+4IK+u4zAuLHvW+dhSo2El21Z2SoEDGZGND6qhVKA9TZL/wKRvi6ahi28N2P4AP+K5EuqAS1vXV6vLErTAcj5rdIVYc3AXyB5ezKaiwh5ASd2m1eWZa63z2pduXVBWPvV8yMa4UwAFFWgZJ+29p8IBUoaqOso0jK9xNN2FjWnDCtdIvlKTlaWQTCzmzCDe2s+KXAxO513+YP7CMiqJflDmY5PFyt88c//WwtiGI+N0mo9+S3DzKJDUcQx0Seik45UahgJyMl4J3RjRiHNdRFeMPOCWzu/yCdK3vAXFmuSmnucg+3fK5zL7V8gXj2YtZPOaEpqznI1S+ZvGJo26ZiIYboQaOaR4nAgH6nUVHwZug6oc4rYxS9Gs/s4LhcFVsibxLVFaaaAHSdFyJskM3cuzxiPsIPpAGBVuhi5Sugs2xMdV8CvRVIjexLv2l0vJMnkyVeVDy7pTf8cY5fveOPthxEN6uH2NlmXKYt5wGiDBAxXanFxQ9rXhFajxeX4hQyjSzhio0kMYj+zVhxKDQww/emfrigHJcth56DYsd8TLZ/rkKKdfbthi6Pgsw85knT+cvmRmbZE0lyG9o6mNUe1kl9xSHZITtw56H4qBV+MyJkjdEeBApuxRogQZFJrI9Wl/M/x22rm2t6rJOrNaB4M5+3Fofe6kT5N0cA/XuDz23i8dXwjP++Z72HbmJ9HzrrMr8wz1KSFwAB93RbTCpGcNZsZSl6i3e4noctLftwAQ4uYydt3RZFvM3hr/vDh5Eh95hEnoRWGQqcpSnNEMPb/pTr0Os0Ls8tvRBPC+OQUXI1ppAqHjezfEPycZS3stLEQMPIOfy9JIX9LymIt4/Ce9QSRIrjC5GsXW3mhkUehX1EBminhQuBaUv4BiCi5pEQ43f1s2kXa28pxVpNofO8EgceBM5ypjckEf7fr9ErgHB311rmM64j2wJRKBsdJRH3uamy9J/KQYxxbjhM69+FcGYEanO4eaZ9DpMnAOvZ8EyCOhJgJnySlrhCrXiisJm2OaeFdnLaD8m0BZx/AbkyUJWPshcMhbrV29OhIRv2V6s3hrQ0VYZkqUAAJcWefsFtQXk/xI5odTPEASCA+DjDdHcHccNWClcH+uTmWRFBmQnJRSbIURm0fj97Zq5I5jeJjyd/ISlH2C5JzkQ72hClMMRuswbd4neYs1a8HTvNWxPaY6y6HXds6p5eTybBqAu+kd/kFWzg8YLcjLQYaKizHopiStsd0IjlQqv11gBbCizXic18brlP+0Uc8f4Zi8v2iJ7i0OnsWodClQ/AcelQ9pW/wVUfj1thbMGP9L0RhUs7mU7PzQYp0JfadNfRtdL3RXg/5AibBPLBGdWTZxt3hVhqMAWLiEFn0CpEyNG2Vos155IViZDGG0bJ8ZPrKkHQusDKV/u2uzdC0Pge9RN3M4Ad7ICmQsM4fgytJw6nRZ8KqsDNawJs2mYDjajGlFahnbv3LZ9osgSe6vf2BMFuCqaBe77lyvDrDb2YBV7+Zszm50lD5Y5GS/T1UdL0USK0Pc2e3Wqv+hxXB3vc6+Dkr3952vVKC2x7fHO03CGukLzbBgGTi0TxQA7KCJukuSNHEHNwYc202zi2VeKywBHgkySUrwPpX0ZpS4FblgoZMvItzkaIMH6ZK++2rqu1fnf1gLGS2kZTgQFFSbGReLkItxQixylJxCngjfAX13o8IyPuvQYTOPEIR8ozNapsFvt93UtsfHwYxCdx5VrYHvmnJe9us7cYHM+XRjVdnZ0xIweSP3B7X+DO6jVc0NEEByaTRdGNrHlLSq+eLW6Oo9PRSRv70Ft3VgEuhcj1VtNzRg1biYYjM1qhYfl9yrGZmz8niKogX+GGQYPcpD3/a364yhHd2z9rX7nqkIZ7+zd/AtpUq5lET4MnPzDlgEAo5VERNixm51F1NxnkK2HLxm3Vtgf0ltwxYpK4M84QWV9LIaH800Q8to5tCy13FuW/y6J2EAaCJz3umJn0+sOziMr7EdlalAT/2B3OrQc5BYc5U1MlayOsZReNucGLav1DlyKYwf3ZDi60k1I/Lox5GdjF+mAYV18LGwojJYLtJiB7n4G4/0L4x3zC2I15jssB5XitTd+R3/fcVEjmmo7UaF5W5AwM1k3cU2PQUTyt4aIvVLJg3BZJXXu1FZcOhim7rKFSqXjQWnJybs2BCffCEw6ENYHQULaBRuk89F3VDTokX48PhDhDfPZ0cEH+uL1OsSmefXBL4zu7wpTl1cGUUv7OASupQLDaG5vpGKEUnxe2hS5ZjHK4GPKbjMo4RLBZNmpfFUI2KcbJ9M3VbBf0t8EtHTkSAlveYXyszwQ9YQ2SEuFKyaZwaAbKEQKfJYuQCkUg4n8KIQemUR457VoKXB+ZiTaG8V4Ec7pG2Js1mVe8UkjdAAAAAAAAAAAAAAAAAAAAAAAADAxMCEwCQYFKw4DAhoFAAQUmfMpJPMwYHYXEj6tZd/zXEzjhJYECMiDWJygUdTuAgIH0AAA");
```

Comment:
Just a nitpick, but I think hex blobs looks better in sourcecode rather than base64.
However, I wonder if this is the right place to put it. Afair, in the Hafas world, we leave authentication data to the apps.